README.md

```diff
@@ -2,10 +2,94 @@
 
 This project provides:
 
-* `AkkaDiscoveryServiceLocator`, a Lagom `ServiceLocator` implementation based on [Akka Discovery][]
-* `AkkaDiscoveryServiceLocatorModule`, a Guice `Module` to wire-up the `AkkaDiscoveryServiceLocator` when using runtime-time DI.
-* `AkkaDiscoveryComponents`, a cake component to wire-up the `AkkaDiscoveryServiceLocator` when using compile-time DI (macwire).
+* [`AkkaDiscoveryServiceLocator`][], a Lagom `ServiceLocator` implementation based on [Akka Discovery][]
+* [`AkkaDiscoveryServiceLocatorModule`][], a Guice `Module` to wire-up the `AkkaDiscoveryServiceLocator` when using runtime-time DI.
+* [`AkkaDiscoveryComponents`][], a cake component to wire-up the `AkkaDiscoveryServiceLocator` when using compile-time DI (macwire).
 
-This is an experimental project. The API and configuration may change.
+Additionally, you will need to choose one of the existing [Akka Discovery][] implementations and configure it for production in your application.
 
+[`AkkaDiscoveryServiceLocatorModule`]: https://github.com/lagom/lagom-akka-discovery-service-locator/blob/v0.0.12/service-locator/javadsl/src/main/java/com/lightbend/lagom/javadsl/akka/discovery/AkkaDiscoveryServiceLocatorModule.java
+[`AkkaDiscoveryServiceLocator`]: https://github.com/lagom/lagom-akka-discovery-service-locator/blob/v0.0.12/service-locator/javadsl/src/main/java/com/lightbend/lagom/javadsl/akka/discovery/AkkaDiscoveryServiceLocator.java
+[`AkkaDiscoveryComponents`]: https://github.com/lagom/lagom-akka-discovery-service-locator/blob/v0.0.12/service-locator/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/akka/discovery/AkkaDiscoveryComponents.scala
```

Comment:
same here. Linking to a fixed version only creates more maintenance. When we release 0.0.13 we need to come here edit this.

Reply:
Same here, stable URLs.